test/support/utils_sycl.h

```diff
@@ -95,18 +95,28 @@ namespace TestUtils
     }
 #endif
 
-#if _ONEDPL_FPGA_DEVICE
-    auto& default_dpcpp_policy = oneapi::dpl::execution::dpcpp_fpga;
+#if ONEDPL_FPGA_DEVICE
     auto default_selector =
-#if _ONEDPL_FPGA_EMU
+#if ONEDPL_FPGA_EMULATOR
         sycl::INTEL::fpga_emulator_selector{};
 #else
         sycl::INTEL::fpga_selector{};
-#endif
+#endif // ONEDPL_FPGA_EMULATOR
+    auto default_dpcpp_policy =
```

Comment:
Why not `auto&`?

Reply:
Which benefits do you see to have `auto&` instead of `auto` here and below?

Reply:
Without & you never test the actual predefined policy objects, always taking a copy first.